client/src/cluster.rs

```diff
@@ -114,7 +114,7 @@ mod tests {
     #[test]
     #[should_panic]
     fn test_cluster_bad_parse() {
-        let bad_url = "httq://my_custom_url.test.net";
+        let bad_url = "http://my_custom_url.test.net";
```

Comment:
This is deliberately a `bad_url` and designed to make the program panic. See `test_cluster_bad_parse()`

Reply:
It's a 🤖

Reply:
lol the bot author needs to make it write better PR titles - 'fix issues' 🤦🏼‍♂️